examples/basic_concept.ipynb

```diff
@@ -0,0 +1,449 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d220666f-dad8-4229-9f24-9d24925b5c25",
+   "metadata": {},
+   "source": [
+    "<h1> 1, Import </h1>\n",
+    "<p>ExeTera utlize HDF5 file format to acquire fast performance when processing the data. Hence the first step of using ExeTera is usually transform the file from other formats, e.g. csv, into HDF5.</p>\n",
+    "<p>ExeTera provides utilities to transform the csv data into HDF5, through either command line or code. </p>\n",
+    "<b>How import works</b>\n",
+    "<p>\n",
+    "a. Importing via the exetera import command:  <br>\n",
+    "<em>\n",
+    "exetera import <br>\n",
```

Comment:
I wouldn't use HTML tags for formatting here when you can use markdown syntax, so # or ## for titles/subtitles, and ``` blocks for literal data like the output from the program.